images/gcloud/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.1
+VERSION = 0.2
```

Comment:
It's possible to do `goversion-gcloudversion` or something. I don't think it's super important though, just look at the git history...